docs/operations/qa_standards.md

```diff
@@ -0,0 +1,60 @@
+# Project Quality Assurance Standards
+This document describes the common practices used to ensure code repositories
+maintain a reasonable standard of quality. This applies to any projects under
+the Neon AI GitHub organization.
+> At the time of writing, this is comprised almost entirely of Python projects,
+  but this document should apply to all projects as much as possible.
+
+## Standard Automated Tests
+All projects should implement the following tests unless there is need for a
+specific exception.
+- `license_tests`: This checks for any incompatibly licensed dependencies. A
```

Comment:
At some point it may be worth decoupling this from pypi, but that's possibly a larger project

Reply:
Ideally, the shared automation would be adapted in this case, so nothing here would change. The workflow file might change some, but the process would be the same